rueppellii-lvlup-asp.net-core/Controllers/AdminController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using rueppellii_lvlup_asp.net_core.DTOs;
+using rueppellii_lvlup_asp.net_core.Extensions;
+using rueppellii_lvlup_asp.net_core.Utility;
+
+namespace rueppellii_lvlup_asp.net_core.Controllers
+{
+    [Route("admin")]
+    [ApiController]
+    public class AdminController : Controller
+    {
+        [HttpPost("add")]
+        [Consumes("application/json")]
+        public IActionResult Add(AddAdminDto addAdminDto)
+        {
+            if (string.IsNullOrEmpty(Request.Headers["usertokenauth"]))
+            {
+                return StatusCode(401, new ErrorMessage("usertokenauth missing"));
+            }
+            if(addAdminDto.IsAnyPropertyNull() || addAdminDto.IsAnyStringPropertyEmpty())
```

Comment:
i can't see the point in this pull request where these 2 methods connecting to AddAdminDto class
don't you need to use UserInputChecker.IsAnyPropertyNull(addAdminDto) instead?